### ISSUE TITLE
Changelog Title Expansion

### ISSUE THREAD

**Issue by StevenLangbroek** [semantic-release/changelog#92]
Hey folks!

We're using Storybook combined with [MDX](https://mdxjs.com/) to host documentation of our design system. We'd like to embed changelogs in our public documentation, but are running into an issue:

`release-notes-generator` doesn't have a template we can use for the "page header" (it's per release best I can tell). What we'd like to do is inject a couple of lines of MDX code on top of the entire page:

```
import { Meta } from '@storybook/addon-docs/blocks';

<Meta title="Core/Changelog />
```

I don't think currently this is possible, since the title property gets prefixed with `# `. Would you be willing to discuss allowing a document-wide header?

**Comment by tommyalatalo** [comment]
I agree with this, even though my use case would be satisfied by just allowing expansion of environment variables into the changelogTitle field.

**Comment by sag1v** [comment]
@StevenLangbroek We are also using `mdx` and importing `Meta`. I've noticed that the `Meta` component creates an element with an `anchor` id which used in the URL, hence the browser scroll into this element.

That means that when `changelog` append new lines on top of it, it will push it down thus the browser will scroll to bottom of the page on load.

I'm trying to figure  out a way to keep these lines on top

```js
import { Meta } from '@storybook/addon-docs/blocks';

<Meta title="MY TITLE />
```

Any idea how can this be achieved?

**Comment by maddesigns** [comment]
we are using a CHANGELOG.md in our root folder and importing the file in a changelog.stories.mdx 

```
import { Meta } from "@storybook/addon-docs/blocks";
import Changelog from "../../CHANGELOG.md";

<Meta title="DesignSystem/Changelog" />

# Changelog

<Changelog />
```

Config for the docs addon:
```
    {
      name: "@storybook/addon-docs",
      options: {
        configureJSX: true,
        babelOptions: {},
        transcludeMarkdown: true,
      },
    },
```
